playbooks/common/openshift-cluster/config.yml

```diff
@@ -17,6 +17,24 @@
       - package_version
       - docker_image_availability
       - docker_storage
+  when: ansible_distribution != "Fedora"
+
+- name: Verify Requirements - Fedora
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - docker_image_availability
+      - docker_storage
+  when: ansible_distribution == "Fedora"
```

Comment:
Alternatively we could set the default required value to a low number, the benefit being preventing situations as described in this RFE https://github.com/openshift/openshift-ansible/issues/4413, that led to checking available space in some other paths that are actively used during the install.

To set the required value, one can set this ansible variable:

```yaml
vars:
    openshift_check_min_host_disk_gb: 0.1
```